src/app/pages/sign-in/sign-in.controller.js

```diff
@@ -1,18 +1,25 @@
-/* eslint-disable no-console */
-function SignInController($scope, authenticationService) {
-  'ngInject';
-  $scope.signIn = () => {
-    $scope.warning = "";
-    const login = $scope.user.email;
-    const password = $scope.user.password;
+'use strict';
 
-    authenticationService.signInToFirebase(login, password)
-    .then(function(response) {
-      if (response) {
-        $scope.warning = response.message;
+
+class SignInController {
+    constructor($scope, authenticationService, toastr) {
+        'ngInject';
+        this.scope = $scope;
+        this.authenticationService = authenticationService;
+        this.toastr = toastr;
+      }
+      
+      $onInit () {
+        this.scope.signIn = () => {
```

Comment:
it is not correct if you use class( put all function to this)

Reply:
Done